Dockerfile.mysql

```diff
@@ -0,0 +1,15 @@
+FROM mysql
```

Comment:
It's better to specify which version to use. If it's the latest, then indicate that.